README.md

```diff
@@ -10,12 +10,29 @@ The goal of this project is to make feature complete Gothic client-app compatibl
 [![Build status](https://ci.appveyor.com/api/projects/status/github/Try/opengothic?svg=true)](https://ci.appveyor.com/project/Try/opengothic)  
 Core gameplay is done, you can complete first chapter, as well as all addon content for any guild. 
 
-##### How to play
+##### Install on Windows
 1. Install original gothic game from CD/Steam/GOG/etc  
 *you have to install original game, since OpenGothic does not have any game assets or game scripts as built-in*
 2. [Download latest stable build](https://github.com/Try/opengothic/releases/latest)
 3. run '/OpenGothic/bin/Gothic2Notr.exe -g "C:\Program Files (x86)\Path\To\Gothic II"'
 
+##### Build it for Linux
+1. Install dependencies:
```

Comment:
I want to simplify this to:
1. Install dependencies:
`cmake g++ gcc-c++ doxygen glslang-tools libvulkan-dev libasound2-dev`

Reply:
I'm not sure it's a good idea. Now that I think about it, I think the opposite should be done: specifying that this is not only for Ubuntu, but for Ubuntu 20.04. The same with openSUSE, for which I used Tumbleweed.

Reply:
Motivation for one list is simplicity. It 3 time easier to maintain one list instead of three.
Another thing, what I'm worry about is approach. Testing on each type of linux is very time consuming, why not just build a package list based on actual dependencies?